src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -666,4 +673,38 @@ public static boolean supportJDBC43(Connection con) throws SQLException {
     public static String escapeSingleQuotes(String name) {
         return name.replace("'", "''");
     }
+    
+    /**
+     * Returns if connected to SQL Azure
+     * @param con
+     *        connection to server
+     * @return boolean
+     * @throws SQLException
+     */
+    public static boolean isSqlAzure(Connection con) throws SQLException {
+        if (_determinedSqlAzureOrSqlServer) {
```

Comment:
where are we setting _determinedSqlAzureOrSqlServer = true? also, this needs to be applied to isSqlAzureDW as well.

Reply:
also you need to set _isSqlAzure